src/app/features/test-record-summary/test-record-summary.component.ts

```diff
@@ -11,4 +11,9 @@ export class TestRecordSummaryComponent {
   @Input() testRecords: TestResultModel[] = [];
 
   constructor() {}
+
+  getTestTypeName(testResult: TestResultModel) {
+    const testNames = testResult.testTypes.map(t => t.testTypeName).join(',');
```

Comment:
return expression here and save lines.

Reply:
Always in two minds about this one for readability but the function name is semantic enough that I feel alright about it